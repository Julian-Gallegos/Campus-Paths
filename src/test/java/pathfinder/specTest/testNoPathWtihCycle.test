# Find a path with Dijkstra's algorithm when
# there is a cycle of nodes in the graph and
# there is no path from starting node to destination node

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A C 0.0
AddEdge simpleGraph C D 0.0
AddEdge simpleGraph D A 1.0

FindPath simpleGraph A B