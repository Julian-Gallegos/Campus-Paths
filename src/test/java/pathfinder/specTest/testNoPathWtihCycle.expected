# Find a path with Dijkstra's algorithm when
# there is a cycle of nodes in the graph and
# there is no path from starting node to destination node

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 0.000 from A to C in simpleGraph
added edge 0.000 from C to D in simpleGraph
added edge 1.000 from D to A in simpleGraph

path from A to B:
no path found